Sources/PublicInterface/Request.swift

```diff
@@ -30,25 +30,33 @@ public class Request {
         self.payload = payload
         self.url = url
     }
+    
+    public static func payloadId() -> Int {
+        let now = Int(NSDate().timeIntervalSince1970) * 1000
+        let datePart: Int = now * Int(pow(Double(10),Double(3)))
+        let extraPart: Int = Int(floor(Double.random(in: 0...1) * Double(pow(Double(10),Double(3)))))
+        let payloadId: Int = datePart + extraPart
+        return payloadId
+    }
```

Comment:
```suggestion
    /// Generates new ID for a request that is compatible with JS clients.
    /// - Returns: new ID
    public static func payloadId() -> RequestID {
        // Port of the payloadId() from the JS library to make sure that
        // generated IDs fit into JS's number size (max integer size is
        // 2^53 - 1).
        //
        // NOTE: new Date().getTime() is time in milliseconds in JS, however,
        // in iOS it is time in seconds. So instead of 1000, we multiply by 1 million
        //
        // NOTE: the id may still be greater than JS max number, but
        // this would happen if the date is in year 2255
        //
        // JS code:
        //    const datePart: number = new Date().getTime() * Math.pow(10, 3)
        //    const extraPart: number = Math.floor(Math.random() * Math.pow(10, 3))
        //    const id: number = datePart + extraPart
        let datePart = Int(Date().timeIntervalSince1970 * 1000_000)
        let extraPart = Int.random(in: 0..<1000)
        let id = datePart + extraPart
        return id
    }
```